apps/dav/lib/Connector/Sabre/Directory.php

```diff
@@ -203,7 +208,7 @@ public function createDirectory($name) {
 	 * @throws \Sabre\DAV\Exception\NotFound
 	 * @throws \Sabre\DAV\Exception\ServiceUnavailable
 	 */
-	public function getChild($name, $info = null) {
+	public function getChild($name, $info = null, Request $request = null, IL10N $l10n = null) {
```

Comment:
```suggestion
	public function getChild($name, $info = null, IRequest $request = null, IL10N $l10n = null) {
```

Any reason the interface is not sufficient?